hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/hevm/HevmPropagatedCallFailure.java

```diff
@@ -41,7 +41,11 @@ public enum HevmPropagatedCallFailure {
     /**
      * The call failed because its externalizing its result would exceed the maximum number of child records.
      */
-    RESULT_CANNOT_BE_EXTERNALIZED(INSUFFICIENT_CHILD_RECORDS);
+    RESULT_CANNOT_BE_EXTERNALIZED(INSUFFICIENT_CHILD_RECORDS),
+    /**
+     * The call failed because invalid fee was submitted for an EVM call
+     */
+    INVALID_FEE_SUBMITTED(CustomExceptionalHaltReason.INVALID_FEE_SUBMITTED);
```

Comment:
Here again inconsistency in qualification.  I personally like having the `CustomExceptionalHaltReason.` qualifier on all these enums, but either way is ok as long as it's consistent.

Oh, whoops, I see there's no choice here due to name clash.  So ... I suggest (mildly) that the other two enum's argument be qualified as well.